### ISSUE TITLE
DLT v2 ?

### ISSUE THREAD

**Issue by Azkali** [PTS93/DLT-PCB-Design-Files#2]
Hi, first of all I wanted to let you know that I followed your work on this DLT and Pi CM4 carrier board and all I can say is thank you for such great work and keeping your sources open !

I am currently working on a few software projects surrounding Jetson platforms ( Jetson Nano, Xavier NX, TX1 ).
And I'd be very interested with the help of our contributors to pursue your work on this DLT.

But I think it would be best to share you an [invitation to our Discord](https://discord.gg/53mtKYt) to discuss this further if you happen to have any time for us.

**Comment by timonsku** [comment]
Hi Azkali,
I somewhat shelved the project atm as I concentrated on some other products. I do plan to continue working on it but might go into a bit of a different direction. I realized in order to continue at this point I need to work towards commercializing and for that I do need to think a bit more about the potential audience. Focusing a little bit on a gaming direction was definitely part of those thoughts.
Making a device like that commercially is a huge undertaking so I'm currently bootstrapping a bit with smaller products.

I really appreciate the interest and it is nice to see that there is an active community around the Jetson line outside of traditional industry. What did you want to talk about specifically?

**Comment by cybik** [comment]
For the record, while I would not be able to help on the development of this thing, if at some point y'all would need early version testers (that would need to pay some non-ridiculous amounts to get the development boards and parts), I may be interested.

**Comment by Azkali** [comment]
Thank you for the answer, we are interested to create a similar device and initially wanted to base ourselves on your work, we wanted to have some insights on BMS and are planning to add support for the different M2 sockets as those are the only important feature that seem to be missing for our simple use case. We are mostly doing software and low level development and thouht a Xavier NX based tablet would be helpful for this purpose especially when we have the ability to swap the SoM for TX2 NX / Nano.

**Comment by timonsku** [comment]
BMS is by far the most complicated part, at least if you are planning for a real world device. It is one of the reasons why I shelved it at the moment as that will need quite a lot of considerations and gets really complicated when you introduce USB-PD which you most certainly want to support with such a high power SoC. If you make it a charging only port it gets a bit easier but if you also want to support USB3.0 and Dual Role port as you would see on most modern tablets it become a project on its own.

Breaking PCI-e out is pretty trivial of course, that would certainly be part of the next iteration.

**Comment by Azkali** [comment]
Yes.. After some thinking we came to this conclusion as well for the BMS. Well dual role in our case would be great to have but USB2.0/3.0 is enough. Are you still keeping your sources open even with the commercial plan ? ( I would buy one to support you anyway )

At the moment is there any issue preventing me from using current board revision granted that I have someone produce it ?

Maybe I could always give help breaking out PCI-e so that you have less to do ? Also could a fund raising / crowdfunding to see how many people are interested might help you ? ( just throwing some ideas here and there, no worries I understand that you have your priorities :) )

**Comment by timonsku** [comment]
Yes the design will always be open source, that would not change. The plan for commercialization was always there but never very concrete.

The current revision has a mistake with the HDMI pinout, the eDP/DP port was working fine it is what I used to test the eDP panel.

When it comes to board layout I don't really need help but thanks :)
The complications are more on a sourcing and system design level. I have some friends who can help me with the BMS side of things but before I put more work into this I need to come up with a more detailed business plan.

A crowdfunding would certainly be coming but that is only really useful once I have a product prototype and know where things are going. You need to price out parts and have suppliers for everything before a crowdfunding can be done otherwise there is a high risk that I loose money.
Right now I'm finishing up Piunora which went through Crowdfunding in April: https://www.crowdsupply.com/diodes-delight/piunora
